composer.json

```
@@ -10,14 +10,16 @@
         }
     ],
     "require": {
+        "php": ">=7.0",
```

```suggestion
        "php": ">=7.3",
```

We can bump PHP version to 7.3 as we plan to release major version

---

Fixed in 9cfa9387.

Either way the minimum version for the code to work (due to `public const`) would be 7.1.